app/src/main/java/org/mozilla/fenix/settings/PairFragment.kt

```diff
@@ -35,13 +39,17 @@ class PairFragment : Fragment(R.layout.fragment_pair), BackHandler {
                 onNeedToRequestPermissions = { permissions ->
                     requestPermissions(permissions, REQUEST_CODE_CAMERA_PERMISSIONS)
                 },
-                onScanResult = { pairingUrl ->
-                    requireComponents.services.accountsAuthFeature.beginPairingAuthentication(
-                        requireContext(),
-                        pairingUrl
-                    )
-                    findNavController(this@PairFragment)
-                        .popBackStack(R.id.turnOnSyncFragment, false)
+                onScanResult = {
+                        pairingUrl ->requireComponents.services.accountsAuthFeature.beginPairingAuthentication(requireContext(),pairingUrl)
+                    val vibrator = context?.getSystemService(Context.VIBRATOR_SERVICE) as Vibrator
```

Comment:
There's an Android KTX function for this that is shorter.
```suggestion
                    val vibrator = requireContext().getSystemService<Vibrator>()
```